docs/networking.md

```diff
@@ -1,4 +1,25 @@
-## Kubernetes Networking Options
+# Kubernetes Networking Options
+
+## Supported networking options
+
+The following table provides the support status for various networking providers with regards to Kops version: 
+
+| Network provider | Experimental | Stable | Deprecated | Removed | 
+| ------------ | -----------: | -----: | ---------: | ------: |
+| AWS VPC | ? | ? | - | - |
+| Calico | 1.6 | 1.11 | - | - |
+| Canal  | 1.12 | - | - | - |
+| Cilium | 1.9 | 1.15 | - | - |
+| Classic | ? | ? | 1.17 | 1.18 |
```

Comment:
Classic has been unsupported as of Kubernetes 1.4.0

Reply:
I think Classic should simply be omitted from the table.

Reply:
Classic is mentioned in the docs, API, and kops help, so we need to handle that as well.

Reply:
I removed it from this doc, at least.